sam_bot_description/src/description/sam_bot_description.urdf

```diff
@@ -279,11 +275,13 @@
   </joint>
 
   <gazebo reference="lidar_link">
-    <sensor name="lidar" type="ray">
+    <sensor name="lidar" type="gpu_lidar">
```

Comment:
I believe this should go into the SDF file now, right? 

Also configured like: https://gazebosim.org/docs/latest/sensors/#lidar-sensor

Reply:
Will be adding this to the SDF file also.

Reply:
Should be good to go with [57db422](https://github.com/ros-navigation/navigation2_tutorials/pull/102/commits/57db4224d143056d692990a95f7cc3ad7a7cb45b). Also it looks like both ``<sensor><ray></ray></sensor>`` and ``<sensor><lidar></lidar></sensor>`` work, but I changed it to ``<ray>`` anyways.